komga/src/main/kotlin/org/gotson/komga/domain/service/SeriesLifecycle.kt

```diff
@@ -209,12 +209,18 @@ class SeriesLifecycle(
     return selected
   }
 
+  fun getThumbnailBytesById(seriesId: String, thumbnailId: String): ByteArray? {
+    return thumbnailsSeriesRepository.findAllBySeriesId(seriesId)
```

Comment:
you should have a new repository method `findById` instead of doing all the work in application code